README.md

```diff
@@ -90,25 +87,39 @@ latest breaking changes in high level are below:
 
 This release also does a lot of dependency updates and relaxations to unblock upstream build issues.
 
-### 4.0.0 - Open Breaking Changes
+# Backends
 
-See tracking issue: [curve25519-dalek/issues/521](https://github.com/dalek-cryptography/curve25519-dalek/issues/521)
+Curve arithmetic is implemented and used by one of the following backends:
 
-# Backends
+| Backend            | Selection | Implementation                                                | Bits / Word sizes |
+| :---               | :---      | :---                                                          | :---              |
+| `serial`           | Automatic | serial software                                               | `32` and `64`     |
+| `fiat`             | Manual    | Formally verified field arithmetic from [fiat-crypto]         | `32` and `64`     |
+| `simd`             | Automatic | Intel AVX2 / AVX512 IFMA accelerated backend                  | `64` only         |
 
-Curve arithmetic is implemented and used by selecting one of the following backends:
+## Automatic Backend Selection
 
-| Backend            | Implementation                                                | Target backends             |
-| :---               | :---                                                          | :---                        |
-| `[default]`        | Automatic runtime backend selection (either serial or SIMD)   | `u32` <br/> `u64` <br/> `avx2` <br/> `avx512` |
-| `fiat`             | Formally verified field arithmetic from [fiat-crypto]         | `fiat_u32` <br/> `fiat_u64` |
+Currently on the intel `x86_64` targets with `64` bit word size, the backend
+selection is automatic between the `simd` and `serial` backends.
 
-To choose a backend other than the `[default]` backend, set the
-environment variable:
+The `simd` backend may be automatically selected if the AVX instruction set is
+detected available on the target CPU.
```

Comment:
1. `AVX` is incorrect; it should be `AVX2`.
2. The wording `may` is unclear. We should precisely specify when each will always be enabled, and when it always be disabled.

Reply:
Hmm.. maybe [f41ab86 works](https://github.com/dalek-cryptography/curve25519-dalek/pull/531/commits/f41ab86da454d6dfa3946771011f441e0f2e9a93) - [Rendered](https://github.com/pinkforest/curve25519-dalek/blob/cleanup-features/README.md) ?